api/src/main/java/org/openmrs/module/openconceptlab/client/OclClient.java

```diff
@@ -42,11 +45,16 @@
 import java.util.zip.ZipInputStream;
 
 public class OclClient {
-	
+
 	public static final String DATE_FORMAT = "yyyy-MM-dd'T'HH:mm:ss'Z'";
-	
+
 	public static final String FILE_NAME_FORMAT = "yyyyMMdd_HHmmss";
-	
+
+	public static final String oclApiUrl = "api.openconceptlab.org";

```

Comment:
Did you look at our naming convention for constants?https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NamingConventions

Reply:
Change oclApiUrl to OCL_API_URL